src/org/jitsi/impl/neomedia/rtp/remotebitrateestimator/RemoteBitrateEstimatorAbsSendTime.java

```diff
@@ -264,23 +276,23 @@ private boolean isBitrateImproving(int newBitrateBps)
      */
     @Override
     public RawPacket reverseTransform(
-            RawPacket packet)
+        RawPacket packet)
     {
-        logger.info("Using RemoteBitrateEstimatorAbsSendTime: Instantiating.");
-
         incomingPacketInfo(System.currentTimeMillis(), absoluteSendTimeEngine
                 .getAbsSendTime(packet), packet.getPayloadLength(),
-                packet.getSSRCAsLong());
+            packet.getSSRCAsLong());
         return packet;
     }
 
-    private void incomingPacketInfo(
+    public void incomingPacketInfo(
```

Comment:
this seems like an odd name.  seems like it should be `processIncomingPacketInfo` or `onIncomingPacketInfo` or something